R/module_transform_data.R

```diff
@@ -74,7 +74,11 @@ ui_transform_teal_data <- function(id, transformators, class = "well") {
 
 #' @export
 #' @rdname module_transform_data
-srv_transform_teal_data <- function(id, data, transformators, modules = NULL, is_transform_failed = reactiveValues()) {
+srv_transform_teal_data <- function(id,
+                                    data,
+                                    transformators,
+                                    modules = NULL,
+                                    is_transform_failed = reactiveValues()) {
```

Comment:
this change is not needed. Theoretically nothing has changed

Reply:
Ah, yes this is a leftover from teal_data_r pushing the line over the length limits on the style. I'll remove it.